.github/workflows/awstest.yml

```diff
@@ -10,9 +10,9 @@ jobs:
     if: github.repository == 'nf-core/mhcquant'
     runs-on: ubuntu-latest
     steps:
+      # Launch workflow using Tower CLI tool action
```

Comment:
Same here, `pipeline: ....` and `revision:....` are no longer needed.

Reply:
Somehow I missed these lines, but it should be fixed now